examples/device/ep/csrc/nixl_ep.cpp

```
@@ -58,6 +59,12 @@ static void sleep_ms(int milliseconds) {
     std::this_thread::sleep_for(std::chrono::milliseconds(milliseconds));
 }
 
+static uint64_t milliseconds_to_cycles(double milliseconds, int device_clock_rate_khz) {
```

Cppcheck (2.20.0)

[information] Include file

(missingIncludeSystem)